index.html

```diff
@@ -198,7 +214,7 @@ <h4>Site map</h4>
                 Contact Us <br>
                 Legal</p>
             <span class="footer__site-map--play">
-                <p>Play online <br>
+                <p>Play online<br>
```

Comment:
Also row 197/213, 198/214 and compare row 16/16 with 102/110 and 87/95 if you like :)